### DISCUSSION TITLE
Spaced Repetition for finding an equilibrium

### DISCUSSION THREAD

**Discussion by CalabashSquash** [open-spaced-repetition/free-spaced-repetition-scheduler#24] [Ideas]
Have there been any use cases for spaced repetition in a context outside of memorisation? I have been thinking about one: spacing out activities that will naturally find some optimal equilibrium in the interval.

For example, physical activities like stretching.

Imagine you are able to touch your toes, but if you were to stop for `n` days, you would lose that ability.

Now let's say you wanted to remind yourself to touch your toes using something like Anki (I don't know the specifics of FSRS but I believe the same applies to that specific algorithm). Your interval would keep increasing upon successful toe-touching. Eventually, however, the interval would become greater than `n` days, causing you to lose the ability, thus causing the interval to be reset to 0 (or <10m or whatever Anki does).

Optimally, an algorithm would find out that the ability gets lost at `~n` days and reach equilibrium at just less than `n`.

A few random thoughts:
- This might get complicated by the fact that "failure" could imply two different consequences. That is to say, if you go over `n` days, do you:
    - completely have to start from scratch and build back up?
    - have the ability to decrease the interval a bit and quickly get back to full ability?
- This probably relies on success not being binary. Something like:
    - "I need to start again"
    - "That was too hard bit not that much too hard"
    - "That was just easy enough"
    - "Too easy"
- This of course is not suitable for, or applicable to, memorisation.

I am sharing this here to get thoughts. I understand it's pretty niche.

Thanks :)

**Comment by L-M-Sherlock** [comment]
FSRS is based on DSR model, which is derived from real data. I don't know whether other activities may follow DSR model.
